pelican/generators.py

```diff
@@ -643,7 +643,7 @@ def generate_context(self):
         self.hidden_pages, self.hidden_translations = (
             process_translations(hidden_pages))
 
-        self._update_context(('pages', ))
+        self._update_context(('pages', 'hidden_pages'))
         self.context['PAGES'] = self.pages
```

Comment:
should probably be removed; it is no where documented that pages is available as `pages` and `PAGES` + it is the (only?) variable I know of that actually is made available as UPPERCASE.

If suggest deprecating `PAGES`, issuing a warning if it is called, and update the default themes to use pages instead of PAGES.

@avaris @justinmayer thoughts?